torch/_torch_docs.py

```diff
@@ -12490,27 +12490,27 @@ def merge_dicts(*dicts):
 add_docstr(
     torch.where,
     r"""
-where(condition, x, y, *, out=None) -> Tensor
+where(condition, input, output, *, out=None) -> Tensor
```

Comment:
wait this should be input and other not output right?

Reply:
Yes, you are right, I've messed up for some unknown reason sorry!
I fixed it in the last commit